src/app/Rules/CreateRule.tsx

```diff
@@ -205,7 +205,7 @@ export const CreateRuleForm: React.FC<CreateRuleFormProps> = (_props) => {
     [setFormData],
   );
 
-  const exitForm = React.useCallback(() => history.push('/rules'), [history]);
+  const exitForm = React.useCallback(() => navigate('/rules'), [navigate]);
```

Comment:
This could probably also be a `navigate('..', { relative: 'path' })`, and there are a couple of other places where relative navigation looks like it would make sense - most places that use `navigate()` would probably be worth re-evaluating.

Reply:
Updated all "exitForm" to go to `..` path.